mountpoint-s3.yaml

```diff
@@ -36,6 +36,16 @@ pipeline:
 
   - uses: strip
 
+subpackages:
+  - name: ${{package.name}}-compat
+    description: "Compatibility package to place binaries in the location expected by upstream helm charts"
+    pipeline:
+      - runs: |
+          mkdir -p ${{targets.contextdir}}/mountpoint-s3/bin
+          ln -sf /usr/bin/mount-s3 ${{targets.contextdir}}/mountpoint-s3/bin/mount-s3
+          cp /usr/lib/libgcc_s.so.1 ${{targets.contextdir}}/mountpoint-s3/bin/libgcc_s.so.1
+          cp /usr/lib/libfuse.so.2 ${{targets.contextdir}}/mountpoint-s3/bin/libfuse.so.2
```

Comment:
from my previous reviews with @EyeCantCU we always symlink shared libraries and don't use cp but I'll wait for RJ to confirm here once again if this is something we can proceed with.

Reply:
yeah its possible though, let me do it real quick.

Reply:
good point, thanks!